### PR TITLE
@stratusjs/idx WIP

### PR DESCRIPTION
**Pull request by apocist**

### CODE REVIEW

File: packages/idx/src/idx.ts [hunk 1]
```diff
@@ -1978,7 +1979,6 @@ Stratus.Services.Idx = [
                         'StreetSuffix',
                         'UnitNumber',
                         'City',
-                        'PostalCode',
```
Comment (chadwickmeyer) [inline packages/idx/src/idx.ts]: @apocist  Why did you remove Postal Code?

File: packages/idx/src/property/admin/search.filter.component.html [hunk 2]
```diff
@@ -9,8 +9,8 @@ <h3>Location</h3>
             <div class="location-specs" data-layout="row">
                 <div class="search-input" data-flex="50" data-flex-gt-sm="25">
                     <md-input-container class="md-block minimal">
-                        <label>City</label>
-                        <input data-ng-model="options.query.City"
+                        <label>City, Neighborhood, Zip</label>
+                        <input data-ng-model="options.query.Location"
```
Comment (chadwickmeyer) [inline packages/idx/src/property/admin/search.filter.component.html]: @apocist actually, I wanted the ADMIN to make you specify the value of specific fields. The admin should not be "smart" it must be precise. Because there are cases where something like "marin" may be both a city and a neighborhood, but they want only the neighborhood. So the admin should not use the generic fields. That's a convenience method for the general public in 95% of use cases it's good enough.

Comment (chadwickmeyer) [reply packages/idx/src/property/admin/search.filter.component.html]: I'll accept this and change it back in my next PR.

Comment (apocist) [reply packages/idx/src/property/admin/search.filter.component.html]: We can have them enter specific fields, but if they don't add it to the Location field, it won't display on the frontend's Location field. (right now we don't have a City field on the front end anymore because it was replaced with Location)
